<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsTabPageForm</class>
 <widget class="QWidget" name="SettingsTabPageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="InfiniteStashFolderLabel">
     <property name="text">
      <string>Infinite Stash Folder</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="Torchlight2SharedStashFolderLabel">
     <property name="text">
      <string>Torchlight 2 Shared Stash</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="Torchlight2SharedStashFolderLineEdit"/>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="InfiniteStashFolderLineEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="Torchlight2SharedStashFolderPushButton">
     <property name="text">
      <string>Set Folder</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="InfiniteStashFolderPushButton">
     <property name="text">
      <string>Set Folder</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
